put_get_test.go

```diff
@@ -386,23 +372,19 @@ func testPutGet(t *testing.T, isStream bool) {
 	gzw := gzip.NewWriter(&b)
 	gzw.Write([]byte(originalContents))
 	gzw.Close()
-	if err = os.WriteFile(fname, b.Bytes(), os.ModePerm); err != nil {
+	if err := os.WriteFile(fname, b.Bytes(), os.ModePerm); err != nil {
 		t.Fatal("could not write to gzip file")
 	}
 
 	runTests(t, dsn, func(dbt *DBTest) {
 		dbt.mustExec("create or replace table " + tableName +
 			" (a int, b string)")
+		defer dbt.mustExec("drop table " + tableName)
```

Comment:
What do you think about replacing defer with this? https://ieftimov.com/posts/testing-in-go-clean-tests-using-t-cleanup/
Of course it'd better to use `drop table if exists` in that manner.

Reply:
I would prefer `t.Cleanup` in most cases, but here I think `defer` is more appropriate because we want these defer statements to execute as soon as `func(dbt *DBTest)` finishes. If we use `t.Cleanup` here, the table we created will remain until the whole test finishes. 

Although it doesn't matter if we use `defer` or `t.Cleanup` here,  the behaviour is still different

https://github.com/snowflakedb/gosnowflake/blob/ed37dbc38c55554609f596670add06c7d04468de/driver_test.go#L314-L330